deployments/tyk/bootstrap.sh

```diff
@@ -6,7 +6,8 @@ deployment="Tyk"
 log_start_deployment
 bootstrap_progress
 
-dashboard_base_url="http://localhost:3000"
+dashboard_base_url="http://tyk-dashboard.localhost:3000"
+portal_base_url="http://tyk-portal.localhost:3000"
 gateway_base_url="http://localhost:8080"
```

Comment:
maybe we can add hostname here for completeness? also, we'll need it to demo custom domain